config/kubernetes-sigs/org.yaml

```diff
@@ -64,9 +64,11 @@ members:
 - lavalamp
 - leakingtapan
 - lichuqiang
+- Lion-Wei
 - Liujingfang1
 - liztio
 - m00nf1sh
+- m1093782566
```

Comment:
oops :)

@Lion-Wei Can you remove the last commit and amend the commit message of the first one? That should solve it.

Reply:
Hi, @nikhita . Sorry, my mistake. I was plan to add both m1093782566 and me in this pr. :slightly_smiling_face:

Reply:
No problem! :smile:

Reply:
Thanks all :)